recipes-bsp/u-boot/u-boot_2021.10.bb

```diff
@@ -0,0 +1,53 @@
+require recipes-bsp/u-boot/u-boot-common.inc
+require recipes-bsp/u-boot/u-boot.inc
+
+PROVIDES += "u-boot"
+DEPENDS += "bc-native dtc-native"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+UBOOT_URL:rzg2l-sbc = "git://github.com/Renesas-SST/u-boot.git"
+SRC_URI:rzg2l-sbc = "${UBOOT_URL};name=machine;protocol=https;branch=${BRANCH}"
+BRANCH:rzg2l-sbc = "scarthgap/rz-sbc"
```

Comment:
It should be styhead branch instead of scarthgap.

Reply:
Updated in the commit: f3f29a38b83f01d83b0fac06eeb1db2d8fa46a55